plugins/jira/skills/status-analysis/data-collection.md

```diff
@@ -0,0 +1,359 @@
+---
+name: Data Collection
+description: Fetching issues, changelogs, and comments from Jira for status analysis
+---
+
+# Data Collection
+
+This module handles all Jira API interactions for the Status Analysis Engine. It fetches issues, their hierarchies, changelogs, and comments.
+
+## Overview
```

Comment:
I’m not sure whether doing it via the jira-doc-generator skill would be sufficient.